src/daria/image/image.py

```diff
@@ -89,21 +120,25 @@ def __init__(
             self.name = "Unnamed image"
             self.timestamp = None
 
-            # TODO this should come through a config file
-            self.colorspace: str = "bgr"
+            if no_colorspace_given:
+                warn("Please provide a colorspace. Now it is assumed to be BGR.")
 
         elif isinstance(img, str):
             pil_img = PIL_Image.open(Path(img))
             self.img = np.array(pil_img)
 
             # PIL reads in RGB format
-            self.colorspace: str = "rgb"
+            self.colorspace: str = "RGB"
+            self.metadata["color_space"] = "RGB"
 
             # Read exif metadata
             self.exif = pil_img.getexif()
-            self.timestamp: datetime = datetime.strptime(
-                self.exif.get(306), "%Y:%m:%d %H:%M:%S"
-            )
+            if self.exif.get(306) is not None:
```

Comment:
Just out of curiosity (I believe this makes much sense): Did you come across an exmaple in which the previous code failed?

Reply:
Yes, haha. I wanted to talk to you about this on friday.. Remind me to ask you on Tuesday, or whenever we talk again (I am in principle free all today and tomorrow as well).

Reply:
> View / edit / reply to [this conversation](https://app.reviewnb.com/pmgbergen/DarIA/pull/57/discussion/) on ReviewNB
> 
> _jwboth commented on 2022-09-24T19:59:39Z_ ----------------------------------------------------------------
> 
> Line #2.    denoised_image.plt_show()
> Try to clean up the jupyter notebook, and not include keyboard interrupts etc. A clean file would be actually be best.
> 
> 
> _EStorvik commented on 2022-09-25T05:46:24Z_ ----------------------------------------------------------------
> 
> I am not sure what happened here... This worked on my laptop, and should not require any keyboard interrupt.

The problem lies in the nature of Jupyter notebooks, I think (also a reason why I do not like them). When executing them, you are changing them, which results in a non-empty git diff. Your local output in the terminal (including when you terminate a run) are stored in the notebook file. This is how these things got online.

Reply:
It looks like when reseting the kernel, a notebook is cleaned.